src/components/OrderForm/index.js

```diff
@@ -0,0 +1,114 @@
+import Link from 'next/link';
+import {useState} from 'react';
+
+import useStore from '../../hooks/useStore';
+import StyledCommentField from '../CommentField/styled';
+import StyledLink from '../Link/styled';
+import StyledOrderOptions from '../OrderOptions/styled';
+import StyledSelectTable from '../SelectTable/styled';
+
+import StyledOrderForm from './styled';
+
+export default function OrderForm() {
+	const [selectOrder, setSelectOrder] = useState(true);
+
+	function handleOrder() {
+		setSelectOrder(!selectOrder);
+	}
+
+	const [isHidden, setIsHidden] = useState(false);
+
+	function handleHidden() {
+		setIsHidden(!isHidden);
+	}
+
+	const [showCheckout, setShowCheckout] = useState(false);
+
+	function handleCheckout() {
+		setShowCheckout(disabled => !disabled);
+	}
```

Comment:
It's certainly not the best way, but it's the only one that worked. The `Radio-Buttons` are disabled, because otherwise there were Bugs. 
Maybe we can look up there tomorrow together?